examples/pca_batch.py

```diff
@@ -33,8 +33,10 @@
 def main(readcsv=read_csv, method='svdDense'):
     infile = "./data/batch/pca_normalized.csv"
 
+    # This is an optional argument z score can be configured differently
```

Comment:
`# 'normalization' is an optional parameter to PCA; we use z-score which could be configured differently`